great_expectations/checkpoint/checkpoint.py

```diff
@@ -407,8 +408,41 @@ def _run_validation(
                 f"Exception occurred while running validation[{idx}] of Checkpoint '{self.name}': {e.message}."
             ) from e
 
+    @public_api
     def self_check(self, pretty_print=True) -> dict:
-        # Provide visibility into parameters that Checkpoint was instantiated with.
+        """
```

Comment:
Could we add an annotation to `pretty_print`?

Reply:
great catch :) Thank you